### PR TITLE
Update Repo content to standard format

### PR DESCRIPTION
**Pull request by BenLloydPearson**
* Convert Readme to standard readme format with focus on Nylas docs content.
* Add Contributing.md
* Add pull_request_template.md
* Rename LICENSE file and fix copyright declaration

### CODE REVIEW

File: Contributing.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+# How To Contribute
+## Report a Bug or Request a Feature
+
+If you encounter any bugs while using this software, or want to request a new feature or enhancement, please [create an issue](https://github.com/nylas/nylas-java/issues) to report it, and make sure you add a label to indicate what type of issue it is.
+
+## Contribute Code
+
+Pull requests are welcome for bug fixes. If you want to implement something new, [please request a feature](https://github.com/nylas/nylas-java/issues) first so we can discuss it.
+
+While writing your code contribution, make sure you follow the testing conventions found in the [tests directory](https://github.com/nylas/nylas-java/tree/master/tests) for any components that you add. We use [codecov](https://codecov.io/gh/nylas/nylas-java) to test coverage, please ensure that your contributions don’t cause a decrease to test coverage.
```
Comment (ddlatham) [inline Contributing.md]: We are still lacking in automated tests and I have not hooked up any code coverage analysis tools.

File: README.md [hunk 2]
```diff
@@ -1,40 +1,55 @@
...
     implementation("com.nylas.sdk:nylas-java-sdk:1.X.X")
 
-For projects using Maven, add the following to your POM file:
+**Install via Maven**: For projects using Maven, add the following to your POM file:
 
     <dependency>
       <groupId>com.nylas.sdk</groupId>
       <artifactId>nylas-java-sdk</artifactId>
       <version>1.X.X</version>
     </dependency>
```
Comment (ddlatham) [inline README.md]: Using the term "Install" feels a bit odd to me for using the SDK as a dependency, but I do see some people use and if that's the preferred term at Nylas, ok.

File: README.md [hunk 3]
```diff
@@ -1,40 +1,55 @@
...
+    
+**Build from source**: To build from source, clone this repo and build the project with Gradle.
 
-## Running examples
-- Check out the repo
-- Create an example.properties file from the template, and enter your nylas application client id and client secret:
-  - `cp src/examples/resources/example.properties.template src/examples/resources/example.properties`
-  - Edit `src/examples/resources/example.properties`
-- Run `gradlew runExample -Pexample=com.nylas.examples.HostedAuthUrlExample` or any other example from `src/examples/java/com/nylas/examples`
```
Comment (ddlatham) [inline README.md]: I think it's helpful to have instructions for running the examples.  Is there a better place for them, do you think?

Comment (BenLloydPearson) [reply README.md]: We can move this info to the Readme for the other examples directory when the examples are reorganized.

File: README.md (hunk 2 above)
Comment (BenLloydPearson) [reply README.md]: I want to keep the top level header as Install, but I changed the subsections to Setup instead.

File: Contributing.md (hunk 1 above)
Comment (BenLloydPearson) [reply Contributing.md]: I've removed this content from the file.

**Pull request merged**